assets/scss/components/_report-error.scss

```diff
@@ -1,8 +1,25 @@
+/*
+Report Error
+
+Display 'Get help with this page' form, and loading spinner while form is loaded by ajax.
+
+Markup:
+<div class="report-error">
+    <a class="report-error__toggle">Get help with this page</a>
+    <div class="report-error__content">
+        <div class="report-error__loading"></div>
+    </div>
+</div>
+
+Styleguide Report Error
```

Comment:
Report error doesn't feel right for this. What's the actual pattern rather what's it's being used for? 

A loading spinner?